src/App.jsx

```diff
@@ -1,107 +1,32 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+const posts = postsFromServer.map(post => ({
+  ...post,
+  user: getUserbyId(post.userId),
+  comments: getCommentsbyId(post.id),
+}));
+
+function getUserbyId(userId) {
+  return usersFromServer.find(
```

Comment:
You should handle the case if the function will find nothing. You can return null instead